packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts

```diff
@@ -33,6 +32,7 @@ import {
 import { getPrivateNetworkName } from '../data/network';
 import { useAllPrivateNetworks } from './useNetwork';
 import { mapPluginsFromArrayToObject } from '../data/plugins';
+import { OidcFormValues } from '@/pages/oidc-provider/AddOrUpdateOIDCProvider.page';
```

Comment:
**nitpick:** if it needs to be reused, define it in "types" folder and use it. `pages` is not the correct folder to have reusable types.